Tools/autotest/common.py

```diff
@@ -821,6 +821,22 @@ def get_disarm_delay(self):
         """Return disarm delay value."""
         raise ErrorException("Disarm delay is not supported by vehicle %s frame %s", (self.vehicleinfo_key(), self.frame))
 
+    def max_rc_channels(self):
+        return 16
+
+    def rc_option_value_for_arming_channel(self):
+        return 104
+
+    def get_arming_channel(self):
+        option = self.rc_option_value_for_arming_channel()
+        for i in range(1, self.max_rc_channels() + 1):
+            v = self.get_parameter("RC%u_OPTION" % i)
+            self.progress("v=%u" % v)
+            if v == option:
+                self.progress("rudder is on channel %u" % i)
+                return i
+        raise PreconditionFailedException("No rudder configured")
```

Comment:
Can we use yaw? I think it's more common for all vehicles than rudder.

Reply:
Yes.